CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 3.7)
 
+cmake_policy(SET CMP0074 NEW)
```

Comment:
Ah, ok. I also tried with ZLIB_ROOT and wondered why it failed in https://ci.appveyor.com/project/tbeu/matio/builds/34974493/job/n8shbn1nor7a3dk8.

Reply:
But it is not a CMake 3.10 nor 3.7 feature.

```
CMake Error at CMakeLists.txt:3 (cmake_policy):
  Policy "CMP0074" is not known to this version of CMake.
```